src/creator.ts

```diff
@@ -723,7 +725,7 @@ export class MSICreator {
         name: 'AutoUpdate',
         key: productRegKey,
         type: 'integer',
-        value: '1',
+        value: '[AUTOUPDATEENABLED]',
```

Comment:
This value will be replaced, right? Or is it okay that `[AUTOUPDATEENABLED]` isn't an integer?

Reply:
Yes, the value will be replaced with whatever the value of the MSI property AUTOUPDATEENABLED is